client/QA/index.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import axios from 'axios'
+
+
+const getData = () =>{
+  console.log('test')
+axios({
+  method: 'get',
+  url: 'https://app-hrsei-api.herokuapp.com/api/fec2/hr-rfe/qa/questions/?page=1&count=100&product_id=5',
+  headers : {'Authorization': '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'}
```

Comment:
Pls set the api token to be a config variable that references your gitignore'd config file